src/ViewModel/ViewSelector.php

```diff
@@ -12,35 +12,45 @@ final class ViewSelector implements ViewModel
     use ArrayAccessFromProperties;
     use ArrayFromProperties;
 
-    private $articleUrl;
+    private $primaryUrl;
+    private $primaryLabel;
     private $jumpLinks;
-    private $figureUrl;
-    private $figureIsActive;
+    private $secondaryUrl;
+    private $secondaryLabel;
+    private $secondaryIsActive;
     private $otherLinks;
     private $sideBySideUrl;
+    /**
+     * @var string
+     */
 
     public function __construct(
-        string $articleUrl,
+        string $primaryUrl,
+        string $primaryLabel,
```

Comment:
`primaryUrl` and `primaryLabel` could become one parameter and be a Link. Same could be done for `secondaryUrl` and `secondaryLabel`.